### PR TITLE
added required permissions to the description and readme

### PR DESCRIPTION
**Pull request by iyeshaya**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
added the required permissions to the description and readme

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender_description.md [hunk 1]
```diff
@@ -22,16 +22,21 @@ In order to use the Cortex XSOAR application, use the default application ID.
 To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal, with mobile and desktop flows enabled.
```
Comment (Itay4) [inline Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender_description.md:22]:
```suggestion
To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal. For more details, follow [Self Deployed Application - Device Code Flow](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#device-code-flow).
```

File: Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender_description.md [hunk 2]
```diff
@@ -22,16 +22,21 @@ In order to use the Cortex XSOAR application, use the default application ID.
...
 
 #### Required Permissions
-* microsoft-365-defender-incidents-list (one of the following):
-    * Incident.Read.All	- Application
-    * AdvancedHunting.Read.All - Application
+* microsoft-365-defender-incidents-list:
```
Comment (Itay4) [inline Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender_description.md:25]: no need to split into commands here, just a list of all permissions

File: Packs/Microsoft365Defender/Integrations/Microsoft365Defender/README.md [hunk 3]
```diff
@@ -2,6 +2,27 @@ Microsoft 365 Defender is a unified pre- and post-breach enterprise defense suit
 prevention, investigation, and response across endpoints, identities, email, and applications to provide integrated
 protection against sophisticated attacks.
 
+## Authentication
+For more details about the authentication used in this integration, see <a href="https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication">Microsoft Integrations - Authentication</a>.
```
Comment (Itay4) [inline Packs/Microsoft365Defender/Integrations/Microsoft365Defender/README.md:6]:
```suggestion
Use the [device code flow](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#device-code-flow)
to link Microsoft 365 Defender with Cortex XSOAR.

To connect to the Microsoft 365 Defender:
1. Fill in the required parameters.
2. Run the ***!microsoft-365-defender-auth-start*** command. 
3. Follow the instructions that appear.
4. Run the ***!microsoft-365-defender-auth-complete*** command.

At the end of the process you'll see a message that you've logged in successfully.

#### Cortex XSOAR App

In order to use the Cortex XSOAR application, use the default application ID.
```9093c354-630a-47f1-b087-6768eb9427e6```

#### Self-Deployed Azure App

To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal. For more details, follow [Self Deployed Application - Device Code Flow](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#device-code-flow).
```

File: Packs/Microsoft365Defender/Integrations/Microsoft365Defender/README.md [hunk 4]
```diff
@@ -2,6 +2,27 @@ Microsoft 365 Defender is a unified pre- and post-breach enterprise defense suit
...
+
+#### Required Permissions
+* microsoft-365-defender-incidents-list:
+    * offline_access - Delegated 
+      
+      And one of the following:
+      * Incident.Read.All	- Application
```
Comment (Itay4) [inline Packs/Microsoft365Defender/Integrations/Microsoft365Defender/README.md:13]:
```suggestion
      * Incident.Read.All - Application
```

File: Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender_description.md (hunk 2 above)
Comment (iyeshaya) [reply Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender_description.md:25]: Done

**Review by Itay4 — APPROVED** [review]

**Pull request merged**